.github/workflows/build-desktop-apps.yml

```
@@ -14,6 +14,11 @@ on:
   pull_request:
     types: [labeled]
   workflow_dispatch:
+    bluetooth:
```

actionlint (1.7.4)

expected "inputs" key for "workflow_dispatch" section but got "bluetooth"

(syntax-check)